refactor, environment and expansion to cover HPCs beyond P1 DTU HPC, …

starostka opened the pull request:
As discussed at the last meeting.

To have a single documentation cover all P1 related HPC facilities (and since I get requests regarding NGC and expect also soon Gefion)...

I've made changes to the docs, re-ordered the sections and also made a few adjustments to the environment (without breaking the traditional `pip` method) to make it easier for me and hopefully others to make changes once cloned.

Sections about NGC and Gefion are not yet that detailed and should be seen as placeholders for now until I have more detailed information - But they'll signal the intent.

@SkafteNicki and @mikkelfo I will let this PR sit so we can discuss it at the next governance meeting. You are welcome to take a look beforehand if you have the time.

starostka committed:
refactor, environment and expansion to cover HPCs beyond P1 DTU HPC, now also NGC and Gefion

mikkelfo reviewed (commented):
Content wise it looks good to me. Once we know better how we're handling the three compute systems, we can always expand on it, but for now it's a decent start I'd say

starostka committed:
explicit docs ordering, merged md files reduced count 11 -> 8

SkafteNicki commented on mkdocs.yaml:
```diff
@@ -74,9 +74,11 @@ extra_css:
-  - Get started: getstarted.md
-  - Compute Resources: compute_resources.md
-  - Learning Resources: learning_resources.md
-  - Support: support.md
-  - Contribute: contribute.md
+  - About Compute Resources: about.md
+  - P1 DTU HPC: 01_p1_dtu_hpc.md
+  - P1 NGC HPC: 02_p1_ngc_hpc.md
+  - P1 Gefion HPC: 03_p1_gefion_hpc.md
```
lets consider a bit of nested structure
```
nav:
  - Home: index.md
  - About Compute Resources:
      - about.md
      - P1 DTU HPC: 01_p1_dtu_hpc.md
      - P1 NGC HPC: 02_p1_ngc_hpc.md
      - P1 Gefion HPC: 03_p1_gefion_hpc.md
```

SkafteNicki reviewed (commented):
Looks good, happy with the splitting into different pages so we can extend in the future with more details for each system.

A mix of comments, overall I think the biggest thing to improve is to make sure the pros and cons of the different systems are more clearly laid out. For example is should be very clear that P1 Gefion is something the researcher at this point will be paying for themself, else we get requests because people think it is free and will then be disappointed.

robvanderg commented:
Thanks, this looks good!, is there a reason connecting through openconnect is not included?, for linux users it seems the most straightforward way (please let me know if I'm wrong about that)

starostka commented on docs/support.md:
```diff
@@ -1,21 +0,0 @@
-official P1-slack. Simiarly if you have general questions regarding compute, including using the P1-cluster, feel free 
-to post those questions in the `#ask-compute` channel.
-
-Institution | Name                                                                              | E-mail
------------ | --------------------------------------------------------------------------------- | ---
-DTU         | [Nicki Skafte Detlefsen](https://orbit.dtu.dk/en/persons/nicki-skafte-detlefsen)  | <nsde@dtu.dk>
-ITU         | [Rob van der Goot](https://pure.itu.dk/en/persons/rob-van-der-goot)               | <robv@itu.dk>
-KU          | [Mikkel Fruelund Odgaard](https://di.ku.dk/ansatte/?pure=da/persons/770606)       | <miod@di.ku.dk>
-AAU         | [Andreas Aakerberg](https://vbn.aau.dk/da/persons/132677)                         | <anaa@create.aau.dk>
-AU          | [Akhil Arora](https://www.au.dk/en/akhil.arora@cs.au.dk)                          | <akhil.arora@cs.au.dk>
```
Yes of course! This was accidentally removed.

starostka committed:
Resolved comments, added warnings indicating waiting list (ngc) and not free (gefion), added pricing model table (not detailed with numbers)

starostka commented:
Incorporated changes based on your comments. I've added warnings to the NGC and Gefion sections indicating that the onboarding process is in the making and Gefion is not free.

starostka committed:
keyboard arrow navigation

starostka committed:
ngc info adjustment

starostka committed:
max 2 gpus by default

starostka committed:
storage request only
